### DISCUSSION TITLE
更新3.25.1后，cloudflare代理返回500

### DISCUSSION THREAD

**Discussion by johncsl** [AlistGo/alist#4969] [General]
更新了3.25.1后，使用cloudflare worker下载会返回500。已测试onedrive和google drive均有这个问题。降级到3.24.0则可以正常下载。

**Comment by anwen-anyi** [comment]
https://github.com/alist-org/alist/issues/4971  已经修复方便的话可以试试测试版能不能用~

**Comment by ypq123456789** [comment]
我也是
